certified-connectors/The IT Tipster/apiProperties.json

```diff
@@ -0,0 +1,22 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "X-API-KEY",
+          "description": "The X-API-KEY for this api",
+          "tooltip": "Provide your X-API-KEY",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
+    "capabilities": [],
+    "publisher": "Shane Cosgrove"
```

Comment:
@theittipster1 for API owners we ask to have the "publisher" and "stackOwner" properties the same. Please see [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiProperties.json).
```
"publisher": "Name of company or organization.",
"stackOwner": "Name of the owner of the service or product."
```